src/GoogleAdWordsEventForwarder.js

```diff
@@ -269,6 +396,31 @@
             return true;
         }
 
+        // Creates a new Consent State Payload based on Consent State and Mapping
+        function generateConsentStatePayloadFromMappings(consentState, mappings, defaults) {
```

Comment:
I see that `generateConsentStatePayloadFromMappings` is always taking the `consentPayloadDefaults` as the third argument.  In that case, you can remove the third argument here, and just use `consentPayloadDefaults` since that's a global variable.